src/cohort-app.js

```diff
@@ -5,6 +5,8 @@
 const express = require('express');
 const bodyParser = require('body-parser')
 const path = require('path')
+const passport = require('passport')
```

Comment:
After a quick google of what the passport module does, does using it mean we could potentially support people signing in with their google or facebook accounts?